worker/provisioner/container_initialisation_test.go

```
@@ -289,9 +294,32 @@ func (s *ContainerSetupSuite) assertContainerInitialised(c *gc.C, ctype instance
 	}
 	s.PatchValue(&provisioner.StartProvisioner, startProvisionerWorker)
 
+	var series string
+	var expected_initial []string
+
+	current_os, _ := version.GetOSFromSeries(version.Current.Series)
```

again, don't swallow errors and don't use _'s